charts/redpanda/templates/servicemonitor.yaml

```diff
@@ -38,10 +38,22 @@ spec:
     scheme: https
     tlsConfig:
       insecureSkipVerify: true
+      ca:
```

Comment:
Only if CA is enabled right? That is an option in the certs stanza of the values file.